### DISCUSSION TITLE
Can anybody explain how do you use symbol search in lazyvim?

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#3095] [Q&A] [answered]
I recently found the shortcut <leader>sS which searches for symbols. But I honestly don't understand how it is working. Firstly, I thought is was something like grep neovim search, but it does not strictly match the string. For example, if I search for "CreateUser" I get result all symbols that just contain "User":
![CleanShot 2024-05-03 at 08 39 58@2x](https://github.com/LazyVim/LazyVim/assets/51330172/de4ec504-8972-4944-8b0e-b4561838c497)

Does anybody know how this shortcut works and how you use it(or why you don't use it)?

**Comment by dusty-phillips** [comment, marked as answer]
It’s querying the go language server for lsp “symbols”. What that means depends on the language server in question. Usually function names, structure types, fields, variable names. I’m not sure why golsp is matching symbols that don’t have update in them. Best guess is that something in your working directory path is named Update, but that doesn’t appear to be the case. 🤔
